core/themes/basis/color/color.inc

```diff
@@ -26,15 +32,122 @@ $info = array(
         'formfocusborder' => '#43afe4',
+
+        'primarytabs' => '#586172',
+        'primarytabstext' => '#010000',
         'buttons' => '#dee2ea',
+
+        'footer' => '#fffeff',
+        'footertext' => '#000100',
+        'footerborder' => '#bbbbbc',
+      ),
+    ),
+    'ice' => array(
+      'title' => t('Ice'),
+      'colors' => array(
+        'header' => '#d0d0d0',
+        'base' => '#999999',
+        'slogan' => '#d0d0d1',
+        'titleslogan' => '#000000',
+        'hovermenu' => '#ffffff',
+        'menutoggle' => '#24272c',
+
+        'bg' => '#ffffff',
+        'text' => '#2a2a2a',
+        'link' => '#00809c',
+        'borders' => '#24272c',
+        'formfocusborder' => '#ffffff',
+
+        'primarytabs' => '#888889',
+        'primarytabstext' => '#d0d0d0',
+        'buttons' => '#014958',
+
+        'footer' => '#d0d0d2',
+        'footertext' => '#000100',
+        'footerborder' => '#24272c',
+      ),
+    ),
+    'plum' => array(
+      'title' => t('Plum'),
+      'colors' => array(
+        'header' => '#4c1c58',
+        'base' => '#000100',
+        'slogan' => '#4c1c59',
+        'titleslogan' => '#ffffff',
+        'hovermenu' => '#9d408e',
+        'menutoggle' => '#2c2c28',
+
+        'bg' => '#fffdf7',
+        'text' => '#301313',
+        'link' => '#9d408d',
+        'borders' => '#2c2c28',
+        'formfocusborder' => '#edede7',
+
+        'primarytabs' => '#2b2b2b',
+        'primarytabstext' => '#301313',
+        'buttons' => '#c9d9e3',
+
+        'footer' => '#e7e7e7',
+        'footertext' => '#000100',
+        'footerborder' => '#2c2c28',
+      ),
+    ),
+    'goldfish' => array(
+      'title' => t('Goldfish'),
+      'colors' => array(
+        'header' => '#e0e4cc',
+        'base' => '#fffeff',
+        'slogan' => '#e0e4cd',
+        'titleslogan' => '#4c5037',
+        'hovermenu' => '#eda766',
+        'menutoggle' => '#4c5036',
+
+        'bg' => '#ffffff',
+        'text' => '#3b3b3b',
+        'link' => '#fa6900',
+        'borders' => '#4c5036',
+        'formfocusborder' => '#f3f4e9',
+
+        'primarytabs' => '#eda766',
+        'primarytabstext' => '#e0e4cc',
+        'buttons' => '#4c5038',
+
+        'footer' => '#798054',
+        'footertext' => '#000100',
+        'footerborder' => '#4c5036',
+      ),
+    ),
+    'mocha' => array(
+      'title' => t('Mocha'),
+      'colors' => array(
+        'header' => '#c63d0f',
+        'base' => '#3b3738',
+        'slogan' => '#c63d0e',
+        'titleslogan' => '#ffffff',
+        'hovermenu' => '#7e8f7d',
+        'menutoggle' => '#dec0b4',
+
+        'bg' => '#faf8f8',
+        'text' => '#3b3738',
+        'link' => '#c63d0f',
+        'borders' => '#dec0b4',
+        'formfocusborder' => '#ffffff',
+
+        'primarytabs' => '#2b2b2b',
+        'primarytabstext' => '#3b3738',
+        'buttons' => '#dec0b4',
+
+        'footer' => '#7e8f7c',
+        'footertext' => '#000100',
+        'footerborder' => '#dec0b4',
```

Comment:
Pink on green, euh. Can we make the border color the same as the footer background? `#6B7D68`

Reply:
looks great now!

Reply:
This border looks better. I'm still not sure people will be able to read either text or the orange link color in the footer for this color pallette, though.